src/base/bittorrent/session.cpp

```diff
@@ -3044,13 +3049,18 @@ void Session::startUpTorrents()
         QByteArray data;
     } TorrentResumeData;
 
-    auto startupTorrent = [this, logger, resumeDataDir](const TorrentResumeData &params)
+    int addCount = 0;
+    const auto startupTorrent = [this, &logger, &resumeDataDir, &addCount](const TorrentResumeData &params)
```

Comment:
Could you describe why? I did not find a reason and can't tell who of you is right because I don't understand the logger well enoght.